x-pack/plugins/observability/server/services/slo/transform_generators/transform_generator.ts

```diff
@@ -40,6 +44,7 @@ export abstract class TransformGenerator {
           },
         },
       }),
+      ...(dataView?.getRuntimeMappings() ?? {}),
```

Comment:
I'm concerned about potential performance issue here. We used to have many runtime fields that we get ride of in previous version.
Here we would always include  _all_ runtime fields of a used data view. It could harm the transform when customers have a lot of mapping, and be unnecessary if they are not using these fields in the queries: https://www.elastic.co/guide/en/elasticsearch/reference/current/transform-limitations.html#transform-runtime-field-limitation